### PR TITLE
Create ravener.json

### PR DESCRIPTION
**Pull request by ravener**
For now a redirect to my vercel blog and discord verification but I look forward to playing around with this domains and I have a vps so I'd probably add more subdomains in the future.

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://ravener.vercel.app

### CODE REVIEW

**Review by iostpa — APPROVED** [review]
seems good to me

File: domains/_discord.ravener.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "ravener",
+    "email": "ravener.anime@gmail.com",
+    "twitter": "RavenRavener",
+    "discord": "292690616285134850"
+  },
+  "records": {
```
Comment (CuteDog5695) [inline domains/_discord.ravener.json:8]: "records" needs to be "record"

**Review by iostpa — CHANGES_REQUESTED** [review]
nvm, file was invalid. check reasons below

**Review by iostpa — APPROVED** [review]
now its fine

**Pull request merged**